TCBApp/TCBApp.WebApi/Extensions/HttpContextExtension.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Net;
+using System.Text.Json;
+
+namespace TCBApp.WebApi.Extensions;
+
+public static class HttpContextExtension
+{
+    public static async Task<T> ConvertByModel<T>(this HttpContext context,HttpWebRequest requestBody)
```

Comment:
Bu method aynan nimadan convert qilib beradi. Requestdanmi yoki Reponsedan?.

Reply:
endi to'g'irlandi